docs/prerequisites.md

```diff
@@ -0,0 +1,48 @@
+# Prerequisites
+
+## Requirements
+
+- Linux or MacOS (Windows isn't supported at the moment).
+- A [Google Cloud](https://console.cloud.google.com) account.
+- [Packer](https://packer.io) to build images
+- Make to use `Makefile` targets
+
+### Setup a Network and Cloud NAT
+
+Google Cloud accounts come with a `default` network which can be found under
+[VPC Networks](https://console.cloud.google.com/networking/networks).
+If you prefer to create a new Network, follow [these instructions](https://cloud.google.com/vpc/docs/using-vpc#create-auto-network).
+
+#### Cloud NAT
+This infrastructure provider sets up Kubernetes clusters using a
+[Global Load Balancer](https://cloud.google.com/load-balancing/) with a public ip address.
+
+Kubernetes nodes, to communicate with the control plane, need to have NAT access or a public ip. By default, the provider creates Machines without a public IP.
+
+To make sure your cluster can communicate with the outside world, and the load balancer, you can create a [Cloud NAT](https://cloud.google.com/nat/docs/overview) in the region you'd like your Kubernetes cluster to live in by following [these instructions](https://cloud.google.com/nat/docs/using-nat#create_nat).
+
+### Create a Service Account
+
+To create and manager clusters, this infrastructure providers uses a service account to authenticate with GCP's APIs.
+
+From your cloud console, follow [these instructions](https://cloud.google.com/iam/docs/creating-managing-service-accounts#creating) to create a new service account with `Editor` permissions. Afterwards, generate a JSON Key and store it somewhere safe.
+
+### Building images
+
+```bash
```

Comment:
- we need to add a caveat here to NOT run the make as `root`
- add a note that this uploads the image as well.

Also they can check if they already have an image using:
`gcloud compute images list --project $GCP_PROJECT --no-standard-images --filter="family:capi-ubuntu-1804-k8s"`

Reply:
Is the filter a prefix here?

Reply:
Because the actual family have the k8s version as suffix https://github.com/kubernetes-sigs/image-builder/blob/master/images/capi/packer/gce/packer.json#L43